src/main/java/com/deusgmbh/xcusatio/ui/profilsettings/ProfileSettings.java

```diff
@@ -1,18 +1,108 @@
+
+import javafx.collections.FXCollections;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+
+/**
+ * 
+ * This is the ProfileSettings class for the User Interface. This class creates
+ * the ProfileSettings tab with all of its components
+ * 
+ * @author Lars.Dittert@de.ibm.com
+ *
+ */
+
+public class ProfileSettings extends FlowPane {
+	private static final String AGE_LABEL = "Alter";
+	private static final String SEX_LABEL = "Geschlecht";
+	private static final String LOCATION_LABEL = "Standort";
+	private static final String CALENDAR_LABEL = "Google-Kalendar";
+	private static final String CALENDAR_BUTTON_LABEL = "Hinzuf\u00fcgen";
+	private static final String PROFILE_SETTINGS_PANE_BORDER_COLOR = "#000000";
+	private static final String AGE_ZERO_TO_NINETEEN = "0-19";
+	private static final String AGE_TWENTY_TO_THIRTYNINE = "20-39";
+	private static final String AGE_FOURTY_TO_FIFTYNINE = "40-59";
+	private static final String AGE_SIXTY_TO_SEVENTYNINE = "60-79";
+	private static final String AGE_EIGHTY_TO_NINETYNINE = "80-99";
+	private static final String DEFAULT_TEXT_TEXTFIELD = "PLZ";
+	private static final String SUBMIT_BUTTON = "Speichern";
+	private static final String TITLE_LABEL = "Profileinstellungen";
+
+	private GridPane gridPane;
+
+	private Label titleLabel;
+	private Label age;
+	private Label sex;
+	private Label location;
+	private Label calendar;
+
+	private ChoiceBox ageChoiceBox;
+	private HBox sexTogglePane;
+
+	private Button calendarButton;
+	private Button submitEditedEntryBtn;
+	private TextField address;
+	private RadioButton male;
+	private RadioButton female;
+
+	public ProfileSettings() {
+		gridPane = new GridPane();
+		sexTogglePane = new HBox();
+
+		this.setStyle("-fx-border-color: " + PROFILE_SETTINGS_PANE_BORDER_COLOR);
+		this.gridPane.setStyle("-fx-border-color: " + PROFILE_SETTINGS_PANE_BORDER_COLOR);
+
+		titleLabel = new Label(TITLE_LABEL);
+		age = new Label(AGE_LABEL);
+		sex = new Label(SEX_LABEL);
+		location = new Label(LOCATION_LABEL);
+		calendar = new Label(CALENDAR_LABEL);
+
+		/* ageChoiceBox */
+		ageChoiceBox = new ChoiceBox(FXCollections.observableArrayList(AGE_ZERO_TO_NINETEEN, AGE_TWENTY_TO_THIRTYNINE,
+				AGE_FOURTY_TO_FIFTYNINE, AGE_SIXTY_TO_SEVENTYNINE, AGE_EIGHTY_TO_NINETYNINE));
+		/* radioButton */
+		ToggleGroup groupRadio = new ToggleGroup();
+		male = new RadioButton();
+		male.setToggleGroup(groupRadio);
+		male.setSelected(true);
+		female = new RadioButton();
+		female.setToggleGroup(groupRadio);
+		sexTogglePane.getChildren().addAll(male, female);
+		/* inputPLZ */
+		address = new TextField(DEFAULT_TEXT_TEXTFIELD);

```

Comment:
Hinter den TextFields etc. fehlen die Logiken. Dahinter steht nämlich nicht immer nur der DEFAULT_TEXT, sondern, wenn es schon ein Profil gibt, also Daten hinterlegt sind, dann sollen diese dort natürlich schon stehen. Dafür benötigst du einen supplier für das Profil. Schau dazu mal in ExcuseEditEntryPane, dort hab ich so einen Supplier registriert, so müsstest du es hier auch machen. Das geht dann zurück zum XCusatioWindow und XCusatio.java initialisiert das dann mit einer Methode aus dem MainController. Falls du das nicht ganz verstehst, komm auf mich zurück, dann mach ich das entweder oder erklärs dir genauer.
In dem Konstruktor für dsa TextField müsste dann bspw stehen profileSupplier.get().getAddress.

Reply:
Außerdem besprechen, ob wir nicht auch Straße und Hausnummer haben wollen

Reply:
Ja Straße und Hausnummer sind auch vorgesehen